src/addons/rails-modal-and-panel/panel.scss

```diff
@@ -20,11 +19,11 @@
 }
 
 .panel__close {
-  float: right;
-  cursor: pointer;
   display: flex;
   padding: var(--rm-space-medium);
   border-radius: 100%;
+  cursor: pointer;
+  float: right;
```

Comment:
@Jeremy-Walton Can we do this without the float? I'd like to avoid the use of float outside of how it relates to interacting with a block of text.

Reply:
This is the old panel code. I'm not changing behavior, just making what we already have compliant with formatting standards.

Rewriting this and the modal code is a completely separate task